specs/confluent-kafka-rest/confluent-kafka-rest.spec

```diff
@@ -0,0 +1,205 @@
+License:            ASL 2.0
+Group:              Development/Tools
+URL:                https://docs.confluent.io/current/kafka-rest/docs/index.html
+
+Source0:            https://github.com/confluentinc/%{pkg_name}/archive/v%{version}.tar.gz
+Source1:            %{pkg_name}.service
+Source2:            %{pkg_name}.init
+Source3:            %{pkg_name}.sysconfig
+Source4:            %{pkg_name}.logrotate
+
+BuildRoot:          %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+Requires:           java >= 1.7.0
+Requires:           confluent-common = %{version}-%{release}
+Requires:           confluent-rest-utils = %{version}-%{release}
+Requires:           logrotate
+%if 0%{?rhel} >= 7
+Requires:           systemd
+%else
+Requires:           kaosv initscripts
+%endif
+
+BuildRequires:      java >= 1.7.0 java-devel >= 1.7.0 maven >= 3.2
+BuildRequires:      confluent-common = %{version}-%{release}
+BuildRequires:      confluent-rest-utils = %{version}-%{release}
+
+Provides:           %{name} = %{version}-%{release}
+
+################################################################################
+
+%description
+The Kafka REST Proxy provides a RESTful interface to a Kafka cluster. It makes
+it easy to produce and consume messages, view the state of the cluster, and
+perform administrative actions without using the native Kafka protocol or
+clients.
+
+################################################################################
+
+%prep
+%setup -qn %{pkg_name}-%{version}
+
+%build
+export JAVA_HOME=$(readlink -f /usr/bin/java | sed "s:/bin/java::")
+
+%install
+rm -rf %{buildroot}
+
+mvn -Dmaven.test.skip=true install
+
+install -dm 755 %{buildroot}%{_bindir}
+install -dm 755 %{buildroot}%{_sysconfdir}/%{pkg_name}
+install -dm 755 %{buildroot}%{_sysconfdir}/sysconfig
+install -dm 755 %{buildroot}%{_sysconfdir}/logrotate.d
+install -dm 755 %{buildroot}%{_datadir}/java/%{pkg_name}
+install -dm 755 %{buildroot}%{_datadir}/doc/%{pkg_name}-%{version}
+install -dm 755 %{buildroot}%{_logdir}/%{pkg_name}
+
+%if 0%{?rhel} >= 7
+install -dm 755 %{buildroot}%{_unitdir}
+%else
+install -dm 755 %{buildroot}%{_initrddir}
+%endif
+
+pushd %{pkg_name}/target/%{pkg_name}-%{version}-package
+  cp -a share/java/%{pkg_name}/*.jar %{buildroot}%{_datadir}/java/%{pkg_name}/
+  cp -a share/doc/%{pkg_name}/* %{buildroot}%{_datadir}/doc/%{pkg_name}-%{version}/
+  cp -a bin/* %{buildroot}%{_bindir}
+  cp -a etc/%{pkg_name}/* %{buildroot}%{_sysconfdir}/%{pkg_name}/
+popd
+
+mv %{buildroot}%{_sysconfdir}/%{pkg_name}/%{pkg_name}.properties \
+   %{buildroot}%{_sysconfdir}/%{pkg_name}/%{pkg_name}.conf
+
+%if 0%{?rhel} >= 7
+install -pm 644 %{SOURCE1} %{buildroot}%{_unitdir}/%{pkg_name}.service
+%else
+install -pm 755 %{SOURCE2} %{buildroot}%{_initrddir}/%{pkg_name}
+%endif
+
+install -pm 644 %{SOURCE3} %{buildroot}%{_sysconfdir}/sysconfig/%{pkg_name}
+install -pm 644 %{SOURCE4} %{buildroot}%{_sysconfdir}/logrotate.d/%{pkg_name}
+
+%check
+mvn test
```

Comment:
Do not execute test while default build process. Look on [this spec](https://github.com/essentialkaos/kaos-repo/blob/master/specs/geos/geos.spec#L119-L121) with the example of how to control test execution with macroses.

Reply:
Fixed (8d618a1).